projects/plugins/protect/src/js/components/firewall-page/styles.module.scss

```
@@ -56,3 +61,38 @@
 .go-back-button {
     margin: calc( var( --spacing-base ) * -1 ) calc( var( --spacing-base ) * -2 ); // -8px | -16px
 }
+.badge {
+    color: var( --jp-gray-30 );
+    font-size: 12px;
+    height: 21px;
+    padding: calc( var( --spacing-base ) / 2 ); // 4px
+    border: 1px solid var( --jp-gray-30 );
+    border-radius: 4px;
+    margin-left: var( --spacing-base ); // 8px
+    margin-bottom: calc( var( --spacing-base ) * 2 ); // 16px
+    display: flex;
+    align-items: center;
+}
+
+.popover {
+    width: 304px;
+    padding: calc( var( --spacing-base ) * 3 ); // 24px
+
+    > .popover-header {
+        display: flex;
+        justify-content: space-between;
+
+        > button {
```

```suggestion
        > button {
```
This works perfectly, I would suggest we use a class name for the button to keep everything consistent and specific.